RUNBOOK.md

```diff
@@ -5,10 +5,8 @@
 0. Take a backup! Run `docker-compose exec backup run_backup.sh` and make sure it succeeded. Also run `git log` and note down the current deployed sha, in case you need to roll back.
 1. `git pull` and `docker-compose build`. This builds and caches the new images locally without interrupting the old server, which reduces downtime.
 2. `docker-compose down` shuts down the server and removes containers. Now downtime has started ticking.
-3. Copy any assets you need from the mediawiki volume `docker-compose cp ...`. Right now this is just images, but we should move everything to static so this step is unnecessary.
-4. `docker volume rm <mediawiki-volume>`, so that the mediawiki container can create a new volume with updates in the next step.
```

Comment:
We still need this step unfortunately.

Reply:
yeah, actually, that step is marked as number 3 now! That's why the diff is weird. 🙂